### DISCUSSION TITLE
Z2M does not work after update...

### DISCUSSION THREAD

**Discussion by bobb16** [Koenkk/zigbee2mqtt#19716] [General]
Hi,
my system is a bananapi m2ultra  with armbian. Z2M seams to start, but does not work after update anymore...
using node v18.12.0 (npm v8.19.2)

`ulf@bananapim2ultra:/opt/zigbee2mqtt$ sudo systemctl status zigbee2mqtt
● zigbee2mqtt.service - zigbee2mqtt
     Loaded: loaded (/etc/systemd/system/zigbee2mqtt.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2023-11-14 22:37:19 CET; 26s ago
   Main PID: 3267 (npm start)
      Tasks: 19 (limit: 2063)
     Memory: 88.2M
        CPU: 26.405s
     CGroup: /system.slice/zigbee2mqtt.service
             ├─3267 npm start
             ├─3280 sh -c node index.js
             └─3281 node index.js

Nov 14 22:37:19 bananapim2ultra systemd[1]: Started zigbee2mqtt.
ulf@bananapim2ultra:/opt/zigbee2mqtt$ sudo systemctl stop zigbee2mqtt
ulf@bananapim2ultra:/opt/zigbee2mqtt$ npm start

> zigbee2mqtt@1.33.2 start
> node index.js

/opt/zigbee2mqtt/lib/extension/publish.ts:22
    philips.tz.effect, // Support Hue effects for groups
               ^
TypeError: Cannot read properties of undefined (reading 'tz')
    at Object.<anonymous> (/opt/zigbee2mqtt/lib/extension/publish.ts:22:13)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/opt/zigbee2mqtt/lib/controller.ts:14:1)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
ulf@bananapim2ultra:/opt/zigbee2mqtt$ 
`
Please help.
Sincerely Ulf.

**Comment by Koenkk** [comment]
try: 'npm ci && npm run build' in the z2m folder

**Reply by bobb16** [reply]
Dear Koen,
thank you very much! But I have done this before with no success. 
In the meantime I moved to the docker container. Played back my backed up config file and it's working again like before. Great! 
The reason for this update was a new sensor I have installed, and which was not supported and still is not.
It's a Tuya ZigBee 3-phase smart meter. 
It's paired:  Zigbee-Modell TS0601, Zigbee-Hersteller _TZE204_x8fp01wi
I'm not an expert... Let's try to figure out, to add this unit. I will see...
Sincerely, Ulf.

**Comment by hemskgren** [comment]
I have the same problem on my updated Debian bullseye on armv7l.

**Reply by bobb16** [reply]
Try the docker container.
